test/shared/src/main/scala/zio/test/ZIOSpec.scala

```diff
@@ -19,7 +19,10 @@ package zio.test
 import zio._
 import zio.stacktracer.TracingImplicits.disableAutoTrace
 
-abstract class ZIOSpec[R <: Has[_]: Tag] extends ZIOApp { self =>
+import org.portablescala.reflect.annotation.EnableReflectiveInstantiation
```

Comment:
Do we need to do this here? I think it is sufficient to do it at the `ZIOSpecAbstract` level.

Reply:
Great catch, works fine when applied only to `ZIOSpecAbstract`